taxi_service/.evn.sample

```diff
@@ -0,0 +1 @@
+SECRET_KEY = 'django-insecure-&*oxc0v9-o%$!m&@0=j9dmz-#u0khk14ym$bs)7nmt5&1f8m#4'
```

Comment:
in a Django project, the .env.sample file should be placed at the project level, which is the same location where you would typically place your .env file. This is generally at the root of your project, alongside the manage.py file.